assets/charts/components/dex/Chart.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v2
```

Comment:
Why can't we use https://github.com/helm/charts/tree/master/stable/dex?

Reply:
it's not maintained and they themselves don't recommend it

Reply:
https://github.com/dexidp/dex/issues/1709

Reply:
If it supports all or most of the feature we need, we could still base on it, even though it's not supported? Has this been checked?

Reply:
No, let me try this then

Reply:
Did you try it?

Reply:
I tried working with the upstream chart, and they have a whole different way of managing certificates, and due to which many changes in deployment have to be made, which is not good because if we want to update it in the future it won't be easy. And that is not compatible with what we need. So I think this is ok for now.